README.md

```diff
@@ -563,6 +563,19 @@ Other Style Guides
     const { left, top } = processInput(input);
     ```
 
+  <a name="destructuring--alias"></a><a name="5.4"></a>
+    - [5.4](#destructuring--alias) Use object alias for getting store the value in the new variable instaed of assigning it to different variable explicitly.
```

Comment:
```suggestion
    - [5.4](#destructuring--alias) Use object alias for getting store the value in the new variable instead of assigning it to different variable explicitly.
```

Reply:
Yes, You are write . I explored the MDN docs where I get this [Assigning to new variable names](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#assigning_to_new_variable_names) for this type of operation .

But I think , this is good add on in the docs because in real life problems we have to destructuring the properties from objects but with the different name

Reply:
Yes, but `const x = obj.y` is often cleaner than `const { x: y } = obj`, is my point.

Reply:
I Think you are write.